flytekit/deck/renderer.py

```diff
@@ -0,0 +1,22 @@
+import typing
+from abc import ABC
+
+import pandas as pd
+
+
+class Renderer(ABC):
+    def render(self) -> str:
```

Comment:
should we call it to_html, that way any pandas-dataframe etc that have to_html method can become Renderable.

```suggestion
    def to_html(self) -> str:
```

Reply:
but don't we want the pandas dataframe rendered to _not_ output actual dataframe values? I'd imagine we want some control over how a particular datatype is rendered.